helmfile/charts/gatekeeper/templates/policies/tests/user-crds.rego

```diff
@@ -1,7 +1,9 @@
-package k8susercrds
+package test.k8susercrds
```

Comment:
Ah the "It works on my computer" issue :sweat_smile: What do you think about leaving this as a separate commit (not PR) when squashing the commits and labeling it as a fix?